scout/server/app.py

```diff
@@ -226,6 +226,11 @@ def spidex_human(spidex):
             return "medium"
         return "high"
 
+    @app.template_filter()
+    def l2fc_2_fc(l2fc: float) -> float:
+        """Converts Log2 fold change to fold change."""
+        return 2**l2fc
```

Comment:
😄 Yes! I have a strong feeling the users would prefer inverted and minus signed values for the < 1x changes (log2 < 0).